temporian/implementation/numpy/data/display_utils.py

```diff
@@ -138,23 +140,19 @@ def display_html(evset: EventSet) -> str:
             row.append(f"{timestamp_repr}")
 
             # Feature values
-            for val, feature in zip(
-                index_data.features[:max_features], visible_feats
-            ):
-                row.append(repr_value_html(val[timestamp_idx], feature.dtype))
+            for feature_idx, feature_schema in enumerate(visible_feats):
+                row.append(repr_value_html(display_features[feature_idx][timestamp_idx], feature_schema.dtype))
+
 
             # Add ... column on the right
             if has_hidden_feats:
                 row.append(ELLIPSIS)
-
-            # Create row and add
-            table.appendChild(html_table_row(dom, row))
-
-        # Add ... row at the bottom
-        if num_timestamps > max_timestamps:
-            # Timestamp + features + <... column if was added>
-            row = [ELLIPSIS] * (1 + len(visible_feats) + int(has_hidden_feats))
+                
+            # Create ellipse row if more than 6 entries
             table.appendChild(html_table_row(dom, row))
+            if timestamp_idx == 2 and num_timestamps > 7:
+                ellipsis_row = [ELLIPSIS] * (1 + len(visible_feats) + int(has_hidden_feats))
```

Comment:
same here - the `2` and `7` need to be computed from `print_max_events`, so that the ellipsis is always in the middle of the first `print_max_events / 2` and the last `print_max_events / 2` events